integrationtest/src/test/resources/jaxbAnnotationProcessorPathsTest/src/main/java/org/mapstruct/itest/jaxb/OrderDetailsDto.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.itest.jaxb;
+
+import java.util.List;
+
+/**
+ * @author Sjaak Derksen
```

Comment:
are you sure I'm the author 😄

Reply:
yes you are, I copied everything from the `jaxbText`. In any case, I got a different idea for this. I'll add some more compilation options such that the current tests are run with `annotationProcessorPaths` and without them.